build/main.mk

```diff
@@ -119,6 +119,10 @@ BINARY_FULL:=$(BINARY_PATH)/$(BINARY_FILE)
 VERSION_SCRIPT:=build/version2code.py
 VERSION_HEADER:=$(BUILD_PATH)/config/Version.ii
 GENERATED_HEADERS:=$(VERSION_HEADER)
+VERSION_LOG:=$(BUILD_BASE)/version.log
+VERSION_INFO=$(strip $(shell grep 'Package version:' $(VERSION_LOG) | awk -F':' '{print $$2}'))
+REVISION_INFO=$(strip $(shell grep 'Revision string:' $(VERSION_LOG) | awk -F':' '{print $$2}'))
+PACKAGE_DETAILED_VERSION=$(VERSION_INFO)-$(REVISION_INFO)
 
 
```

Comment:
Also build/main.mk was probably once copied from the openMSX version. Can you please check how the things you're trying to do here are done in openMSX, and if needed and possible, make it (more) similar?